src/components/DisplayTodo.js

```diff
@@ -1,16 +1,30 @@
 // eslint-disable-next-line
 import React from 'react';
+// eslint-disable-next-line
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// eslint-disable-next-line
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'; // <-- import styles to be used
 
-const DisplayTodo = () => {
+const DisplayTodo = (props) => {
+  const { onHandleCheckBoxChange, onDeleteHandler } = props;
   const todos = JSON.parse(localStorage.getItem('todos'));
```

Comment:
In your App.js file, you passed the list of todos as props to the DisplayToDo component. So, instead of getting the list from the local storage, you can get it from the props, like this: 
` const { onHandleCheckBoxChange, onDeleteHandler, todos } = props;`